src/tool_cfgable.h

```diff
@@ -298,6 +298,7 @@ struct OperationConfig {
   struct State state;             /* for create_transfer() */
   bool rm_partial;                /* on error, remove partially written output
                                      files */
+  bool mptcp; /* enable MPTCP support */
```

Comment:
I don't get their code style here, maybe best to align the comment with the one above?

```c
  bool rm_partial;                /* on error, remove partially written output
                                     files */
  bool mptcp;                     /* enable MPTCP support */
```